uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -198,6 +201,10 @@ define([
           canSubmit = canSubmit && !_.isEqual(initialLeaveRequestAttributes, this.request.attributes());
           // user has added a comment
           canSubmit = canSubmit || this.request.comments.length !== initialCommentsLength;
+          // user has added file Attachment
+          canSubmit = canSubmit || (this.request.files.length + this.request.fileUploader.queue.length) !== initialFilesLength;
+          // did user remove file
+          canSubmit = canSubmit || this.userRemovedFile;
```

Comment:
it will enable/disable the save button , if that is not a requirement then we can ignore it